packages/app/load.envs.js

```diff
@@ -0,0 +1,55 @@
+const { config } = require('dotenv');
+const { resolve } = require('path');
+const { readFileSync } = require('fs');
+const retus = require('retus');
+
+function getVersion() {
+  const packageJson = JSON.parse(
+    readFileSync(resolve(__dirname, './package.json')).toString()
+  );
+  return {
+    version: packageJson.version,
+  };
+}
+
+function getEnvName() {
+  if (process.env.NODE_ENV === 'production') {
+    return '.env.production';
+  }
+  if (process.env.NODE_ENV === 'test') {
+    return '.env.test';
+  }
+}
+
+function getEthFuelL1Contracts() {
+  if (process.env.NODE_ENV === 'development') {
+    const { body } = retus('http://localhost:8080/deployments.local.json', {
+      json: true,
+    });
+
+    return body;
+  }
+}
+
+// Load from more specific env file to generic ->
+[getEnvName(), '.env'].forEach((envFile) => {
+  if (!envFile) return;
+  config({
+    path: resolve(__dirname, envFile),
+  });
+});
+
+// Export the port to be used on vite server and
+// make it accessible to the playwirght tests
+process.env.PORT = process.env.NODE_ENV === 'test' ? 3001 : 3000;
+
+// Export the version to be used on database
+// and application level
+const versions = getVersion();
+process.env.VITE_APP_VERSION = versions.version;
+
+// Export ETH Fuel contracts addresses
+const ethFuelContracts = getEthFuelL1Contracts();
+if (ethFuelContracts && ethFuelContracts.FuelMessagePortal) {
+  process.env.VITE_ETH_FUEL_MESSAGE_PORTAL = ethFuelContracts.FuelMessagePortal;
```

Comment:
oh wait I guess we do add this to the env nvm

Reply:
yeah so in dev we get com L1 ETH docker we run. for test we will use a fixed one from goerli network(.env.example)